lib/Parser.js

```diff
@@ -299,6 +299,11 @@ function parseHeader(message, data, headerStart, headerEnd)
         message.event = parsed;
       }
       break;
+    // allow multiple call info headers
+    case 'call-info':
+      message.addHeader(headerName, headerValue);
+      parsed = 0;
```

Comment:
Why `parsed = 0;` here?

Reply:
`parsed = 0;` to indicate a `true` function return (same way as used
in the default case block).